<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Management</class>
 <widget class="QWidget" name="Management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>688</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>674</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="4">
    <widget class="QPushButton" name="btn_back">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>80</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_back{
background-image: url(:/按键_上一页2.png);
border:none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>50</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="1" column="7">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">
font: 16pt &quot;Ubuntu&quot; ;
color:rgb(255,255,255);
background:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string>LPSFG</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">
font: 16pt &quot;Ubuntu&quot; ;
color:rgb(255,255,255);
background:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string>888ZHU</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">
font: 16pt &quot;Ubuntu&quot; ;
color:rgb(255,255,255);
background:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string>XUXIN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">
font: 16pt &quot;Ubuntu&quot; ;
color:rgb(255,255,255);
background:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string>空白</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">
font: 16pt &quot;Ubuntu&quot; ;
color:rgb(255,255,255);
background:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string>123456</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="8">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-image: url(:/按键_普通用户.png);
border:none;
}
QPushButton:checked{
background-image: url(:/按键_普通用户_按下.png);
border:none;
}
QPushButton:disabled{
background-image: url(:/按键_未定义用户.png);
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-image: url(:/按键_普通用户.png);
border:none;
}
QPushButton:checked{
background-image: url(:/按键_普通用户_按下.png);
border:none;
}
QPushButton:disabled{
background-image: url(:/按键_未定义用户.png);
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="pushButton_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-image: url(:/按键_普通用户.png);
border:none;
}
QPushButton:checked{
background-image: url(:/按键_普通用户_按下.png);
border:none;
}
QPushButton:disabled{
background-image: url(:/按键_未定义用户.png);
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">
font: 16pt &quot;Ubuntu&quot; ;
color:rgb(255,255,255);
background:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string>XIAOMING</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 16pt &quot;Ubuntu&quot; ;
color:rgb(255,255,255);
background:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string>XYZ123</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">
font: 16pt &quot;Ubuntu&quot; ;
color:rgb(255,255,255);
background:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string>空白</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-image: url(:/按键_普通用户.png);
border:none;
}
QPushButton:checked{
background-image: url(:/按键_普通用户_按下.png);
border:none;
}
QPushButton:disabled{
background-image: url(:/按键_未定义用户.png);
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="pushButton_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-image: url(:/按键_普通用户.png);
border:none;
}
QPushButton:checked{
background-image: url(:/按键_普通用户_按下.png);
border:none;
}
QPushButton:disabled{
background-image: url(:/按键_未定义用户.png);
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-image: url(:/按键_普通用户.png);
border:none;
}
QPushButton:checked{
background-image: url(:/按键_普通用户_按下.png);
border:none;
}
QPushButton:disabled{
background-image: url(:/按键_未定义用户.png);
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-image: url(:/按键_普通用户.png);
border:none;
}
QPushButton:checked{
background-image: url(:/按键_普通用户_按下.png);
border:none;
}
QPushButton:disabled{
background-image: url(:/按键_未定义用户.png);
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>94</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-image: url(:/按键_普通用户.png);
border:none;
}
QPushButton:checked{
background-image: url(:/按键_普通用户_按下.png);
border:none;
}
QPushButton:disabled{
background-image: url(:/按键_未定义用户.png);
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btn_del">
     <property name="minimumSize">
      <size>
       <width>145</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_del{
border-image: url(:/按键_删除.png);
border:none;
}
QPushButton#btn_del:pressed{
background-image: url(:/按键_删除_按下.png);
border:none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btn_add">
     <property name="minimumSize">
      <size>
       <width>145</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_add{
border-image: url(:/按键_创建.png);
border:none;
}
QPushButton#btn_add:pressed{
background-image: url(:/按键_创建_按下.png);
border:none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="5">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="btn_fast_login">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:5px solid rgb(44,86,191);
    border-top-left-radius:15px;
    border-top-right-radius:15px;
    border-bottom-left-radius:15px;
    border-bottom-right-radius:15px;
font: 25pt &quot;Microsoft YaHei UI&quot;;
color:rgb(44, 89, 191);
background-color:rgb(0,0,0)
</string>
     </property>
     <property name="text">
      <string>快速登录该用户</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>btn_back</zorder>
  <zorder>btn_add</zorder>
  <zorder>btn_del</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>verticalSpacer_2</zorder>
  <zorder>verticalSpacer_4</zorder>
  <zorder>horizontalSpacer_6</zorder>
  <zorder>horizontalSpacer_7</zorder>
  <zorder>horizontalSpacer_8</zorder>
  <zorder>btn_fast_login</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
